src/app/components/UploadPrescriptionComponents/MedicineItem.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import moment from 'moment';
+import { customTheme } from '../../../../constants/themeConstants';
+import { useNavigation } from '@react-navigation/native';
+import CollapsibleItem from '../../../../components/CollapsibleItem';
+
+const MedicineItem = ({ item, handleDelete, isDetailsComplete, formatDays }) => {
+    const navigation = useNavigation();
+    
+    return (
+        <View key={item.id} className="p-3 border-b border-darkSecondary">
+            <View className="flex-row items-center justify-between">
+                <View className="flex-row items-center">
+                    <Image
+                        className="h-12 w-12 rounded-full"
+                        source={require("../../assets/wellness_product.jpeg")}
+                    />
+                    <View className="ml-4">
+                        <Text className="text-lg font-[appfont-semi]">
+                            {item.name}
+                        </Text>
+                    </View>
+                </View>
+                <View className="flex-row">
+                    <TouchableOpacity
+                        onPress={() => {
```

Comment:
Why are we using onpress = () => {}, just declare a function at the top of the screen so we know what it is doing.